lib/grid_search.py

```diff
@@ -42,18 +42,47 @@ def train(self):
         saved and the hyperparameters that produced the best test error are returned
         """
         best_error = numpy.inf
```

Comment:
Do you think we can call this accuracy instead? All what is "conceptually dual" of error, since you're using 1.0 - error